packs/wp-rocket.js

```diff
@@ -0,0 +1,29 @@
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. Remove Unused CSS, Load JavaScript deferred are names of the features */
+  'render-blocking-resources': "Enable [Remove Unused CSS](https://docs.wp-rocket.me/article/1529-remove-unused-css) and [Load JavaScript deferred](https://docs.wp-rocket.me/article/1265-load-javascript-deferred) in `WP Rocket` to address this recommendation. These features will respectively optimize the CSS and JavaScript files so that they don't block the rendering of your page.",
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. Minify CSS Files is a name of the feature */
+  'unminified-css': "Enable [Minify CSS files](https://docs.wp-rocket.me/article/1350-css-minify-combine) in `WP Rocket` to fix this issue. Any spaces and comments in your site's CSS files will be removed to make the file size smaller and faster to download.",
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. Minify JavaScript Files is a name of the feature */
+  'unminified-javascript': "Enable [Minify JavaScript files](https://docs.wp-rocket.me/article/1351-javascript-minify-combine) in `WP Rocket` to fix this issue. Empty spaces and comments will be removed from JavaScript files to make their size smaller and faster to download.",
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. `Imagify` is an image optimization add-on */
+  'uses-optimized-images': "Enable `Imagify` from the Image Optimization tab in `WP Rocket` and run Bulk Optimization to compress your images.",
+  /** Additional description of a Lighthouse audit for a third-party framework called `WP Rocket`. This is displayed after a user expands the section to see more. No character length limits. Prefetch DNS Requests, Enable CDN are names of the features */
+  'uses-rel-preconnect': "Use [Prefetch DNS Requests](https://docs.wp-rocket.me/article/1302-prefetch-dns-requests) in `WP Rocket` to add \"dns-prefetch\" and speed up the connection with external domains. Also, `WP Rocket` automatically adds \"preconnect\" to [Google Fonts domain](https://docs.wp-rocket.me/article/1312-optimize-google-fonts) and any CNAME(S) added via the [Enable CDN](https://docs.wp-rocket.me/article/42-using-wp-rocket-with-a-cdn) feature.",
```

Comment:
please wrap attributes in backticks, this hides from from our translators. can remove the quotes too.